README.md

```diff
@@ -1,114 +1,128 @@
 # mev-inspect-py
-> illuminating the dark forest 🌲💡
 
-**mev-inspect-py** is an MEV inspector for Ethereum
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+[![Discord](https://img.shields.io/discord/755466764501909692)](https://discord.gg/7hvTycdNcK)
+
+[Maximal extractable value](https://ethereum.org/en/developers/docs/mev/) inspector for Ethereum, to illuminate the [dark forest](https://www.paradigm.xyz/2020/08/ethereum-is-a-dark-forest/) 🌲💡
 
 Given a block, mev-inspect finds:
 - miner payments (gas + coinbase)
 - tokens transfers and profit
 - swaps and [arbitrages](https://twitter.com/bertcmiller/status/1427632028263059462)
 - ...and more
 
-Data is stored in Postgres for analysis
+Data is stored in Postgres for analysis.
 
-## Running locally
-mev-inspect-py is built to run on kubernetes locally and in production
+## Install
 
-### Install dependencies
+mev-inspect-py is built to run on kubernetes locally and in production.
 
-First, setup a local kubernetes deployment - we use [Docker](https://www.docker.com/products/docker-desktop) and [kind](https://kind.sigs.k8s.io/docs/user/quick-start)
+### Dependencies
 
-If using kind, create a new cluster with:
-```
-kind create cluster
-```
+- [docker](https://www.docker.com/products/docker-desktop)
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [helm](https://helm.sh/docs/intro/install/)
 
-Next, install the kubernetes CLI [`kubectl`](https://kubernetes.io/docs/tasks/tools/)
+### Set up
 
-Then, install [helm](https://helm.sh/docs/intro/install/) - helm is a package manager for kubernetes
+Ceate a new cluster with:
 
-Lastly, setup [Tilt](https://docs.tilt.dev/install.html) which manages running and updating kubernetes resources locally
```

Comment:
Probably should keep the instructions to setup Tilt.

Reply:
+1

Reply:
On yes, nice catch. Are you ok with moving this link to the dependencies section?